challenge.js

```diff
@@ -42,7 +42,7 @@
  */
 
 // ANSWER
-
+// document.body.child('D').child('E')
```

Comment:
A,B,C, etc are just names that we're using in the question to refer to those elements - they aren't tag names, classes, or ids. Additionally, the question was asking for a CSS selector rather than JavaScript. One correct answer might be `body > div > p`.